Vagrantfile

```diff
@@ -10,6 +10,7 @@ patroni_nodes = 'p1', 'p2'       # you must have 2+ patroni nodes
 
 Vagrant.configure(2) do |config|
 
+    vip_ip     = IPAddr.new(start_ip)
     next_ip    = IPAddr.new(start_ip).succ
     host_ip    = (IPAddr.new(start_ip) & "255.255.255.0").succ.to_s
```

Comment:
host_ip is not used afterwards. is it a leftover from the PAF vagrant file ?

Reply:
indeed. Can you add a specific commit in this branch to clean it please?

Reply:
will do